File: Java/프로그래머스/0/181877. 대문자로 바꾸기/대문자로 바꾸기.java
class Solution {
    public String solution(String myString) {
        char ch[] = myString.toCharArray();
        for(int i=0; i<ch.length;i++){
            if(ch[i]>=97 && ch[i]<=122)
            ch[i]= Character.toUpperCase(ch[i]);
        }
        String answer= String.valueOf(ch);
        return answer;
    }
}

File: JavaScript/프로그래머스/0/181877. 대문자로 바꾸기/대문자로 바꾸기.js
function solution(myString) {
    var answer = '';
    for(var i=0; i<myString.length; i++){
        if(myString.charAt(i)>'Z' || myString.charAt(i)<'A'){
            answer+=myString.charAt(i).toUpperCase();
        }
        else {
            answer+=myString.charAt(i);
        }
    }
    return answer;
}